### PR TITLE
Further mining changes

### PR DESCRIPTION
**Pull request by thgvr**
## About The Pull Request
Removes ORM from technology nodes. ORM will still be available for now as loot, or on existing ships.
Stops autolathes from accepting ore, it must be smelted.
This should only be merged AFTER https://github.com/shiptest-ss13/Shiptest/pull/2446.

## Changelog

:cl:
del: Ore Redemption Machine is no longer in the "Basic Mining" technology node.
/:cl:

### CODE REVIEW

**Review by Erikafox — APPROVED** [review]
do remove it from all ships (for the moment) too.

File: code/modules/overmap/objects/outpost/outpost_types.dm [hunk 1]
```diff
@@ -84,6 +84,7 @@
 	elevator_template = /datum/map_template/outpost/elevator_test
 	// Uses "test" hangars.
 
+/* THGVR TODO: ADD ORE REFINING TO NANOTRASEN ASTEROID OUTPOST, AND REMOVE THIS COMMENT
```
Comment (MarkSuckerberg) [inline code/modules/overmap/objects/outpost/outpost_types.dm]: can't believe they still haven't removed this comment in 2027

**Review by MarkSuckerberg — APPROVED** [review]

**Comment by thgvr** [comment]
This should ONLY be merged after https://github.com/shiptest-ss13/Shiptest/pull/2446.

**Comment by Xylarr** [comment]
my heart is shattered

**Comment by A-Medster** [comment]
on one hand, this PR is a really fucking cool idea. having to head to the Outpost for more reasons than buying shit. more reasons to interact with other ships
on the other hand, in practice this really really makes rounds drag, 
you now have all these ores, but literally nothing to do with them till you head to the outpost unless you decide to Ghetto Smelt them (AKA tossing it into lava/fire and putting it out afterwards). which tends to hit bigger ships/Ships that land and dont move for a while because they physically *cant* go to the Outpost, they're trying to get things done. Ruins, RP events, etc.
It also removes most good forms of organizing your minerals outside of the Ore Silo (something that's harder to come by with the Departmental Lathes/Circuit printers) and Crate shelves
i know that ORMs being (sorta) outright gone is a temp thing but my god does it hurt to lack one now. it would be nice if this PR made the more complex version of the Ore Smelting process avalable for printing. like the one on the Outpost where you have to use conveyors to move things from the Box to the Smelter to the Autostacker

**Comment by thgvr** [comment]
This won't be merged for some time, and I'll probably go through and set up smelters on ships that deserve it. Need to make all of the machinery not as jank and constructable first, though

**Comment by thgvr** [comment]
I'll think about this again after I can improve the smelter machines further.

**Pull request closed without merging**